README.md

```diff
@@ -32,6 +32,31 @@ Additionally, note that SCOPFLOW and SOPFLOW with HiOp solver use Ipopt to solve
 
 Details installation instructions are given at [INSTALL.md](./INSTALL.md) for information on acquiring, building and installing ExaGO.
 
+If you are a developer with access to the project, we also provide public binaries that are generated through our GitHub actions workflows documented in [README.md](.github/workflows/README.md), and with documentation about usage in the packages section of our repository. Check out a short (< 60s demo) of pulling down a version of ExaGO:
+
+[![asciicast](
```

Comment:
perhaps add a code snippet of the commands you paste in the video, and a brief description of each line :)

Reply:
Part of the charm of asciicast is that you can copy commands, but I still agree :)

Reply:
Check out new command description